woocommerce/woocommerce-android
# [Manual Taxes #1] Analytics events

Analytics events for Manual Taxes:
1. Tap on question mark to show the educational dialog -> "order_taxes_help_button_tapped" 
2. Tap on Edit Tax Rates in Admin button on educational dialog -> "tax_educational_dialog_edit_in_admin_button_tapped"

Closes: #9668


### Testing instructions

Verify the above events are tracked, when:
1. Clicking on "?" icon in order creation 
2. Clicking "Edit Tax Rates in Admin" in the Tax information dialog opened in step 1.

### Images/gif

<img width="1663" alt="Screenshot 2023-08-25 at 22 16 16" src="https://github.com/woocommerce/woocommerce-android/assets/4527432/a7119699-4cde-4a13-8d07-4c3acf0b1c2c">
